mimic/model/clb_objects.py

```diff
@@ -578,12 +579,26 @@ def add_node(self, node_list, lb_id):
                 for node in nodes:
                     node.status = "OFFLINE"
 
+            self._add_node_created_feeds(nodes)
+
             self._verify_and_update_lb_state(
                 lb_id, current_timestamp=current_timestamp)
             return {"nodes": [node.as_json() for node in nodes]}, 202
 
         return not_found_response("loadbalancer"), 404
 
+    def _add_node_created_feeds(self, nodes):
+        """
+        Add "Node created..." feed for each given nodes
```

Comment:
Should this document the fact that there was a private email explaining this format, perhaps, since there's no public doc for it?

Also, `nodes` could use a `:param:` :).